### PR TITLE
MKS - Getting source IP behind LB - New annotation for version >= 1.31

### PR DESCRIPTION
**Pull request by btbenjamin**
Add new annotation  : For Managed Kubernetes Service version >= 1.31

### CODE REVIEW

File: pages/public_cloud/containers_orchestration/managed_kubernetes/getting-source-ip-behind-loadbalancer/guide.en-us.md [hunk 1]
```diff
@@ -176,8 +176,11 @@ Copy the next YAML snippet in a `patch-ingress-controller-service.yml` file:
 
 ```yaml
 metadata:
-  annotations:
-    service.beta.kubernetes.io/ovh-loadbalancer-proxy-protocol: "v2"
+    annotations:
+      # For Managed Kubernetes Service version < 1.31
```
Comment (Lucasgranet) [inline pages/public_cloud/containers_orchestration/managed_kubernetes/getting-source-ip-behind-loadbalancer/guide.en-us.md:186]:
```suggestion
      # For K8S service based on "Loadbalancer for K8S" (default LB provisioned with MKS < 1.31)
```

File: pages/public_cloud/containers_orchestration/managed_kubernetes/getting-source-ip-behind-loadbalancer/guide.en-us.md [hunk 2]
```diff
@@ -176,8 +176,11 @@ Copy the next YAML snippet in a `patch-ingress-controller-service.yml` file:
...
+      service.beta.kubernetes.io/ovh-loadbalancer-proxy-protocol: "v2"
+      # For Managed Kubernetes Service version >= 1.31
```
Comment (Lucasgranet) [inline pages/public_cloud/containers_orchestration/managed_kubernetes/getting-source-ip-behind-loadbalancer/guide.en-us.md:188]:
```suggestion
      # For K8S service based on the Octavia implementation (default LB provisioned with MKS >=1.31)
```

File: pages/public_cloud/containers_orchestration/managed_kubernetes/getting-source-ip-behind-loadbalancer/guide.en-us.md [hunk 3]
```diff
@@ -238,7 +241,10 @@ controller:
   service:
     externalTrafficPolicy: "Local"
     annotations:
+      # For Managed Kubernetes Service version < 1.31
```
Comment (Lucasgranet) [inline pages/public_cloud/containers_orchestration/managed_kubernetes/getting-source-ip-behind-loadbalancer/guide.en-us.md:250]:
```suggestion
      # For K8S service based on "Loadbalancer for K8S" (default LB provisioned with MKS < 1.31)
```

File: pages/public_cloud/containers_orchestration/managed_kubernetes/getting-source-ip-behind-loadbalancer/guide.en-us.md [hunk 4]
```diff
@@ -252,7 +258,10 @@ controller:
...
```
Comment (Lucasgranet) [inline pages/public_cloud/containers_orchestration/managed_kubernetes/getting-source-ip-behind-loadbalancer/guide.en-us.md:267]:
```suggestion
      # For K8S service based on "Loadbalancer for K8S" (default LB provisioned with MKS < 1.31)
```

File: pages/public_cloud/containers_orchestration/managed_kubernetes/getting-source-ip-behind-loadbalancer/guide.en-us.md [hunk 5]
```diff
@@ -238,7 +241,10 @@ controller:
...
       service.beta.kubernetes.io/ovh-loadbalancer-proxy-protocol: "v2"
+      # For Managed Kubernetes Service version >= 1.31
```
Comment (Lucasgranet) [inline pages/public_cloud/containers_orchestration/managed_kubernetes/getting-source-ip-behind-loadbalancer/guide.en-us.md:252]:
```suggestion
      # For K8S service based on the Octavia implementation (default LB provisioned with MKS >=1.31)
```

File: pages/public_cloud/containers_orchestration/managed_kubernetes/getting-source-ip-behind-loadbalancer/guide.en-us.md [hunk 6]
```diff
@@ -252,7 +258,10 @@ controller:
...
```
Comment (Lucasgranet) [inline pages/public_cloud/containers_orchestration/managed_kubernetes/getting-source-ip-behind-loadbalancer/guide.en-us.md:269]:
```suggestion
      # For K8S service based on the Octavia implementation (default LB provisioned with MKS >=1.31)
```

**Review by Lucasgranet — CHANGES_REQUESTED** [review]
Hello

Thank you for your contribution.

I made some reviews to let the user to chose the right annotation.
Currently, a MKS cluster < 1.31 can deploy an Octavia LoadBalancer (using the right annotation to select between "LoadBalancer for K8S" or "Octavia").

**Comment by btbenjamin** [comment]
Hello @Lucasgranet ,

Thanks for your review !
I suggest to add this card too :

image

**Review by Y0Coss — APPROVED** [review]

**Review by Lucasgranet — APPROVED** [review]

**Comment by Y0Coss** [comment]
Thank you for your contribution @btbenjamin

**Pull request merged**